index.ts

```diff
@@ -1,5 +1,6 @@
 import { NextConfig } from 'next';
 import { NextWebVitalsMetric } from 'next/app';
+export { log } from './logger';
```

Comment:
Should we split out the webvitals part into a file as well? Is it time for a src folder? 🤔

Reply:
I have done that now. how do we tell npm that it now should use the output from `dist/`? is it by by chaning the `main` key in package.json or it has more to do with the publishing part?

Reply:
Yes, I think changing `main` and `types` should suffice.

Reply:
changed both to start with dist/